spec/features/user_can_create_glossary_entries_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sign_up_helper'
+require 'create_glossary_entry_helper'
+
+feature 'Glossary creation' do
+  context 'user can create glossary term' do
+    let(:user) { build(:user) }
+    let(:glossary) { build(:glossary) }
+
+    scenario 'with valid inputs will save a new glossary entry' do
+      sign_up(user)
+      create_glossary_entry(glossary)
+      current_term = Glossary.find_by(term: glossary.term)
+      expect(current_path).to eq "/glossaries/#{current_term.id}"
```

Comment:
I wouldn't worry too much about checking the path. It's causing you to query the DB and actually has little value. Why does the user care about the URL? They only care about where they are and what they can see. There are better ways of inferring this information.